src/components/tab-panel/tab-panel.scss

```diff
@@ -1,15 +1,20 @@
-:host(limel-tab-panel) {
-    display: block;
-}
+@import '../../style/internal/variables';
 
-.tab-panel {
+.tab-panel { // contains both limel-tab-bar and tab-content
     height: 100%;
+    position: relative;
     display: flex;
     flex-direction: column;
 }
 
 .tab-content {
-    height: 100%;
-    overflow: auto;
-    flex: 1;
+    position: absolute;
```

Comment:
Also, if we do it this way it seems you are forced to set a height on the tab-panel as a consumer when using it or it will not be usable, which I'm not a fan of :slightly_frowning_face:

Reply:
Related point: we should perhaps make it configurable whether the tab panel should have a fixed height (or max-height, or whatever way we choose to go about it) and scroll any overflow inside it, or have the panel height extend to show all content, like it does now. It might be that other consumers of the code might want or need the page to scroll rather than the tab content. (Assuming it's not a very complicated thing to achieve, but it doesn't seem to me like it would be?)

Reply:
Previously it would not scroll at all, it would just grow with it's content. It should probably still do that if you do not set a height, but if you do then the content should be scrollable

Reply:
I think we mean the same thing :slightly_smiling_face: